frontend/src/app/projects/models/diagrams/model-diagram-dialog/model-diagram-dialog.component.ts

```diff
@@ -25,15 +27,20 @@ import {
   DiagramMetadata,
   ModelDiagramService,
 } from 'src/app/projects/models/diagrams/service/model-diagram.service';
+import { environment } from 'src/environments/environment';
 
 @Component({
   selector: 'app-model-diagram-dialog',
   templateUrl: './model-diagram-dialog.component.html',
   styleUrls: ['./model-diagram-dialog.component.css'],
 })
-export class ModelDiagramDialogComponent {
+export class ModelDiagramDialogComponent implements OnInit {
   diagramMetadata?: DiagramCacheMetadata;
   diagrams: Diagrams = {};
+  cachedUsername: string | null = null;
+  passwordValue: string =
+    'You can insert your project token here using the "Insert token" button. This will invalidate any old token for your user';
```

Comment:
I would make this here possibly undefined (i.e., `passwordValue?: string`) and then use the following in the html:
`{{ passwordValue !== undefined ? passwordValue : 'You can insert your project token here using the "Insert token" button. This will invalidate any old token for your user' }}`. In doing so, we can remove the `isButtonClicked` variable here as we can simply use the state of the `passwordValue` variable.